charts/mlrun-ce/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mlrun-ce
-version: 0.7.1-rc9
+version: 0.7.1-rc90
```

Comment:
`helm lint` fails with `0.7.1-rc10`, because it considers it as below `0.7.1-rc9`. This is because semver requires a dot before the numeric part for it to be considered separately. Setting version to `0.7.1-rc90` until PR is ready, to avoid CI failing.